CLAUDE.md

```
@@ -0,0 +1,629 @@
++
+where:
+
+`<my_product_database_certificates_secrets>`:: Specifies the name of the certificate secret.
+`<ca_certificate_key>`:: Specifies the CA certificate key.
+`<tls_private_key>`:: Specifies the TLS private key.
+`<tls_certificate_key>`:: Specifies the TLS certificate key.
+```
+
+**For YAML files or multiple lines of code**
+* Use a bulleted list to describe the structure of a sample YAML file or explain multiple lines of code in a code block.
+**  List the explanations in the order in which they appear in the code block.
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)